onnxruntime/python/tools/tensorrt/perf/benchmark.py

```diff
@@ -995,9 +998,9 @@ def run_onnxruntime(args, models):
             # Set environment variables for ort-trt benchmarking 
             if "ORT-TRT" in ep:
                 os.environ["ORT_TENSORRT_FP16_ENABLE"] = "1" if "Fp16" in ep else "0"
-                os.environ["ORT_TENSORRT_ENGINE_CACHE_ENABLE"] = "1"
                 os.environ["ORT_TENSORRT_MAX_WORKSPACE_SIZE"] = "4294967296"
-           
+                if args.enable_cache:
```

Comment:
I think we should add a backlog task to use ep options instead of environment variables for configuring TensorRT EP provider options.
see tests as example https://github.com/microsoft/onnxruntime/blob/9a3be9b46adb8ada41a759229935ba30cb233267/onnxruntime/test/python/onnxruntime_test_python.py#L117

Reply:
Agreed, that would make it more robust to future changes to ep options as well.